README.md

```diff
@@ -49,8 +49,23 @@ const BigNumber = require('bignumber.js');
 
 ES6 module:
 
+```bash
+$ npm install bignumber.js --save
+```
+
+```javascript
+import BigNumber from "bignumber.mjs"
```

Comment:
This is a bare import specifier, and it simply won't work, in node or the browser.